crates/cxx-qt-build/src/lib.rs

```diff
@@ -497,8 +497,11 @@ impl CxxQtBuilder {
             builder.flag_if_supported("/std:c++17");
             builder.flag_if_supported("/Zc:__cplusplus");
             builder.flag_if_supported("/permissive-");
+            builder.flag_if_supported("/bigobj"); 
```

Comment:
`cargo fmt` is failing here because there is a space after the `;`

Reply:
amended